cookbook-connector/src/main/java/org/mule/modules/cookbook/CookbookConnector.java

```diff
@@ -97,23 +80,21 @@ public void execute(List<Recipe> recipes) throws Exception {
     }
 
     /**
-     * Description for create
-     *
-     * {@sample.xml ../../../doc/cook-book-connector.xml.sample cook-book:create}
+     * Create Recipes and Ingredients
+     * <p/>
+     * {@sample.xml ../../../doc/cookbook-connector.xml.sample cookbook:create}
      *
-     * @param type Type of the entity
-     * @param entity
-     *            Ingredient to be created
-     * @return return Ingredient with Id from the system.
-     *
-     * @throws SessionExpiredException
-     * @throws InvalidEntityException
+     * @param type   Type of the entity
+     * @param entity Ingredient/Recipe to be created
+     * @return return Id of the entity from the system.
+     * @throws SessionExpiredException Exception thrown when an action is taken by a client who's session has expired.
+     * @throws InvalidEntityException  Exception thrown when an wrong entity or type is mapped to the entity parameter.
      */
     @SuppressWarnings("unchecked")
     @OAuthProtected
     @Processor
     @OnException(handler = CookbookHandler.class)
-    @ReconnectOn(exceptions = { SessionExpiredException.class })
+    @ReconnectOn(exceptions = {SessionExpiredException.class})
```

Comment:
ReconnectOn is present in all message operations, may be it would makes sense to move this at Connector class level.